production/loki-mixin-compiled-ssd/dashboards/loki-writes-resources.json

```diff
@@ -323,7 +331,7 @@
                   "thresholds": [ ],
                   "timeFrom": null,
                   "timeShift": null,
-                  "title": "Memory (go heap inuse)",
+                  "title": "Memory (go heap inuse), ingester",
```

Comment:
I don't think this is correct. `go_memstats_heap_inuse_bytes` will be for the whole process, which in the case of `loki-write` is more than just the ingester.

Reply:
yep the changes, other than the single `ingester` -> `ingester.*` at the top of writes-resources.libsonnet which generates this file were unnecessary so I've backed them out